src/components/Book.js

```diff
@@ -0,0 +1,10 @@
+/* eslint react/destructuring-assignment: 0 */
```

Comment:
### Optional
- [ ] It would be better to fix the ESLint errors instead of using ESLint inline rules

Reply:
I searched a lot and didn't found any specific solution to deal with this Eslint issue,but I will try to fix it next milestones.

Reply:
In the next milestone, don't disable eslint errors, try to fix it without disabling them.